src/Widgets/PricingWidget/PricingWidgetComponent.js

```diff
@@ -1,11 +1,16 @@
 import * as React from "react";
 import * as Scrivito from "scrivito";
 
+import "./pricingWidget.scss";
```

Comment:
Please use `CamelCase.scss` here (and of cause change the name of the original file)

```suggestion
import "./PricingWidget.scss";
```